sebs/aws/container.py

```
@@ -0,0 +1,79 @@
+        username, password, registry_url = self.config.resources.ecr_repository_authorization(
+            self.client
+        )
+
+        try:
+            self.docker_client.login(username=username, password=password, registry=registry_url)
+            super().push_image(repository_uri, image_tag)
+            self.logging.info(f"Successfully pushed the image to registry {repository_uri}.")
+        except docker.errors.APIError as e:
+            self.logging.error(f"Failed to push the image to registry {repository_uri}.")
+            self.logging.error(f"Error: {str(e)}")
+            raise RuntimeError("Couldn't push to Docker registry")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)